yarn-audit-known-issues

```diff
@@ -14,6 +14,7 @@
 {"type":"auditAdvisory","data":{"resolution":{"id":1762,"path":"ngx-md>prismjs","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"1.23.0","paths":["ngx-md>prismjs"]}],"id":1762,"created":"2021-06-28T18:33:51.912Z","updated":"2021-06-28T18:35:48.942Z","deleted":null,"title":"Regular Expression Denial of Service","found_by":{"link":"","name":"Anonymous","email":""},"reported_by":{"link":"","name":"Anonymous","email":""},"module_name":"prismjs","cves":["CVE-2021-32723"],"vulnerable_versions":"<1.24.0","patched_versions":">=1.24.0","overview":"In `prismjs` before 1.24.0 some languages are vulnerable to Regular Expression Denial of Service (ReDoS).\n\n### Impact\n\nWhen Prism is used to highlight untrusted (user-given) text, an attacker can craft a string that will take a very very long time to highlight. Do not use the following languages to highlight untrusted text.\n\n- ASCIIDoc\n- ERB\n\nOther languages are __not__ affected and can be used to highlight untrusted text.\n\n### Patches\nThis problem has been fixed in Prism v1.24.\n\n### References\n\n- PrismJS/prism#2774\n- PrismJS/prism#2688\n","recommendation":"Upgrade to version 1.24.0 or later","references":"- [CVE](https://nvd.nist.gov/vuln/detail/CVE-2021-32723)\n- [GitHub Advisory](https://github.com/advisories/GHSA-gj77-59wh-66hg)\n","access":"public","severity":"high","cwe":"CWE-400","metadata":{"module_type":"","exploitability":7,"affected_components":""},"url":"https://npmjs.com/advisories/1762"}}}
+{"type":"auditAdvisory","data":{"resolution":{"id":1763,"path":"@hmcts/media-viewer>socket.io-client>socket.io-parser","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"3.3.1","paths":["@hmcts/media-viewer>socket.io-client>socket.io-parser"]}],"id":1763,"created":"2021-06-30T16:56:54.859Z","updated":"2021-06-30T16:58:03.642Z","deleted":null,"title":"Resource exhaustion in socket.io-parser","found_by":{"link":"","name":"Anonymous","email":""},"reported_by":{"link":"","name":"Anonymous","email":""},"module_name":"socket.io-parser","cves":["CVE-2020-36049"],"vulnerable_versions":"<3.3.2 || =3.4.0","patched_versions":">=3.3.2 <3.4.0 || >=3.4.1","overview":"The `socket.io-parser` npm package before versions 3.3.2 and 3.4.1 allows attackers to cause a denial of service (memory consumption) via a large packet because a concatenation approach is used.","recommendation":"Upgrade to versions 3.3.2, 3.4.1 or later","references":"- [CVE](https://nvd.nist.gov/vuln/detail/CVE-2020-36049)\n- [GitHub Advisory](https://github.com/advisories/GHSA-xfhh-g9f5-x4m4)\n","access":"public","severity":"high","cwe":"CWE-400","metadata":{"module_type":"","exploitability":7,"affected_components":""},"url":"https://npmjs.com/advisories/1763"}}}
```

Comment:
Can you rebase now, as I believe this has already been merged into master and so you shouldn't have to include this line.

Reply:
Has this been merged into `feature/release2-wa2`